scenarios/pubsub_simple.erl

```diff
@@ -0,0 +1,288 @@
+-module(pubsub_simple).
+
+-behavior(amoc_scenario).
+
+-include_lib("exml/include/exml.hrl").
+-include_lib("escalus/include/escalus.hrl").
+
+-define(HOST, <<"localhost">>). %% The virtual host served by the server
+-define(PUBSUB_ADDR, <<"pubsub.localhost">>).
+-define(WAIT_FOR_STANZA_TIMEOUT, 10000).
+-define(GROUPNAME, <<"pubsub_test_group_name">>).
+-define(DELAY_BETWEEN_MESSAGES, 100).
+-define(WAIT_FOR_NODES, 10000).
+
+-export([init/0]).
+-export([start/1]).
+
+-required_variable({'NUMBER_OF_NODE_CREATORS', <<"Specifies how many users will create node"/utf8>>}).
+-required_variable({'PUBLISHERS_RATIO', <<"Specifies the number of publishers per 10 users"/utf8>>}).
+
+-spec init() -> ok.
+init() ->
+    init_metrics(),
+    ok.
+
+-spec start(amoc_scenario:user_id()) -> any().
+start(Id) ->
+    pg2:create(?GROUPNAME),
+    CreatorsNumber = amoc_config:get('NUMBER_OF_NODE_CREATORS', 3),
+    Client = connect_amoc_user(Id),
+    lager:info("~p~p", [Id, Client]),
+    case assign_role(Id, CreatorsNumber) of
+        creator ->
+            creator_start(Client);
+        publisher ->
+            publisher_start(Client);
+        subscriber ->
+            subscriber_start(Client)
+    end.
+
+%% --- Metrics ---------------------------------------------------------------------------------------------
+
+init_metrics() ->
+    Counters = [ node_create_success, node_create_failed, subscribe_failed,
+                 publish_failed, subscribe_success, publish_success ],
+    Times = [ node_create_time, subscribe_time, publish_time, message_ttd ],
```

Comment:
Since all metrics measuring exec time of sth are prefixed with `[amoc, tide]` I think there is no need to add the `_time` suffix. Additionally I suggest to change counter and time metric names a bit like below (this is not complete list, I imagine you can figure out how to name other metrics based on that):
* `node_creation`
* `node_subscription`
* `item_publication`

I'm aware that this will break your current dashboards, on the other hand I'd like to see these names improved sooner than later. You can also refuse to change because of the existing work you already did. Just let me know :)

Reply:
We would have to adjust the dashboard to the new naming convention (https://github.com/esl/MongooseIM/pull/2193) anyway